.pre-commit-config.yaml

```
@@ -26,12 +32,18 @@ repos:
           - --config-file=pyproject.toml
         additional_dependencies:
           - pytest
-        exclude: "{{cookiecutter.project_slug}}/(src/{{cookiecutter.package_name}}|tests)"
+        exclude: |-
+          (?x)(
+            {{cookiecutter.project_slug}}/(src/{{cookiecutter.package_name}}|tests)
```

```suggestion
            {{cookiecutter.project_slug}}/src/{{cookiecutter.package_name}}
            {{cookiecutter.project_slug}}/tests
```
Since there's the nice new regex sytax, I think it's a lot clearer splitting this on to two lines?

---

Yeah, fair enough, I had wondered about this